public/worker.js

```diff
@@ -0,0 +1,48 @@
+var CACHE_NAME = 'intreped-fe';
+var urlsToCache = [
+  '/',
```

Comment:
I read your slack message, and this is complete supposition, but I think this is the area we would need to look into for caching our videos in the future. Way to go tackling this, I still need to pick your brain on it.